### DISCUSSION TITLE
How to reset fingerprint? Or how does it generate?

### DISCUSSION THREAD

**Discussion by FeiFei-SS** [rustdesk/rustdesk#6566] [Q&A]
I install a Windows 10 in VM(Vitrual Machine), and I copy the full disk create another one. I found these two VM's fingerprint are same, I can only control one VM. I want to know how to solve this problem, thanks!
<img width="629" alt="image" src="https://github.com/rustdesk/rustdesk/assets/45664529/cad8a4a2-f86c-4320-8ec2-cfdf1845c406">
<img width="636" alt="image" src="https://github.com/rustdesk/rustdesk/assets/45664529/0bdff67b-45f3-4e65-ac8d-e5371e26830e">

**Comment by DMSnz** [comment]
If you have cloned your windows install one to one, you would need to reset one of the VMs, machine SID's.

**Reply by FeiFei-SS** [reply]
I already try it, but it didn't change

**Comment by daniel-sjodin** [comment]
https://xkcd.com/979/

We have the same problem. Did you find any solution?

**Reply by daniel-sjodin** [reply]
I found a way that worked for me. 
Change the BIOS UID . I use Proxmox and it looks like this:
![image](https://github.com/user-attachments/assets/f6abe8ad-6b75-4ea6-912d-4ccdba385a5d)

Boot up the machine and Delete the `MachineGuid` entry
In an elevated PowerShell:
`reg delete HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Cryptography /v MachineGuid /f`

Reboot the machine.
